python/iiboost/booster.py

```diff
@@ -22,7 +22,8 @@
 import numpy as np
 import ctypes
 
-from exceptions import RuntimeError
+if sys.version_info[0] < 3:
+    from exceptions import RuntimeError
```

Comment:
Does this even need to be imported at all? FWICT this is already available on Python 2.7 and 2.6 without importing.